src/Apps/Conversation/Components/DetailsHeader.tsx

```diff
@@ -54,8 +63,16 @@ export const DetailsHeader: FC<DetailsHeaderProps> = props => {
 
 export const DetailIcon: React.FC<DetailsProps> = props => {
   const { showDetails, setShowDetails } = props
+  const isXL = __internal__useMatchMedia(themeProps.mediaQueries.xl)
+
+  // When opened, DetailsSidebar is positioned on top of the messages (on XL
+  // layouts only). In those cases, DetailsHeader is redundant with DetailIcon
+  if (showDetails && isXL) {
```

Comment:
I wish we had easy access to breakpoint props on the JS code side, but alas 😢 

One thing we can do to get around this (which sucks a bit DX wise) is:

```tsx
<Media at="xs">{renderXS()}</Media>
<Media at="greaterThanXS">{renderGreaterThanXS()}</Media>
```

Reply:
Using Media sounds good. I'll give it a try now!